app/views/pages/components/categories-listing/listed-category.ejs

```diff
@@ -1,16 +1,5 @@
-<% if (category && category.imageUrl) { %>
-    <div 
-        class='listed-category'
-        style="background-image: url(/articles/image/<%=category.imageUrl%>) !important"
-        onclick="location.href='/?category=<%=category.key%>'">
-        <div class='listed-category__label'>
-            <%=category.displayValue %>
-        </div>
-    </div>
-<% } else if(category) { %>
-    <div class='listed-category' onclick="location.href='/?category=<%=category.key%>'">
-        <div class='listed-category__label'>
-            <%=category.displayValue %>
-        </div>
-    </div>
-<% } %>
\ No newline at end of file
+
```

Comment:
Does this still work if category is invalid? You're not verifying category anymore